java/ql/src/experimental/semmle/code/java/frameworks/Jsf.qll

```diff
@@ -0,0 +1,34 @@
+/**
+ * Provides classes and predicates for working with the Java Server Faces (JSF).
+ */
+
+import semmle.code.java.Type
+
+/**
+ * The JSF class `FacesContext` for processing HTTP requests.
```

Comment:
```suggestion
 * The JSF class `ExternalContext` for processing HTTP requests.
```

Reply:
Thanks for correcting the typo. I've made the change.